nucypher/policy/conditions/evm.py

```diff
@@ -47,19 +47,23 @@ def _resolve_chain(chain: Union[str, int]) -> Tuple[str, int]:
             chain_id, chain_name = pair
             return chain_name, chain_id
     else:
-        raise Exception(f'{chain} is not a known blockchain.')
+        raise ReencryptionCondition.InvalidCondition(
+            f"{chain} is not a known blockchain."
+        )
 
 
 def _resolve_abi(standard_contract_type: str, method: str, function_abi: List) -> List:
     """Resolves the contract an/or function ABI from a standard contract name"""
     if not (function_abi or standard_contract_type):
         # TODO: Is this protection needed?
-        raise ValueError(f'Ambiguous ABI - Supply either an ABI or a standard contract name.')
+        raise ReencryptionCondition.InvalidCondition(
+            f"Ambiguous ABI - Supply either an ABI or a standard contract name."
```

Comment:
Do you think it's too inflexible in list specifically supported standards in this error message?

Reply:
you mean something like:

```suggestion
            f"Ambiguous ABI - Supply either an ABI or a standard contract name (ERC20, ERC721, ERC1155)."
```

Reply:
Yes, exactly.

Reply:
Although, it will be more maintainable if we have a single constant somewhere that is used both for the error message and the actual validity check.

Reply:
Yeah, read my mind. I was thinking of having a set with the standard contract types. We don't have that currently.

There is some validation work we need to eventually do on the Schemas, and having the set of values will help with validation of that field.